client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -109,7 +109,7 @@ <h3 class="oui-header_4 oui-tile__title" data-translate="domain_dashboard_genera
                                                         data-translate="domain_dashboard_enable_web_hosting"></strong>
                                                 <span class="fa fa-info-circle" aria-hidden="true"
                                                       data-uib-tooltip="{{:: 'domain_configuration_web_hosting_offer_START_explain' | translate }}"
-                                                      data-uib-tooltip-append-to-body="true"
+                                                      data-tooltip-append-to-body="true"
```

Comment:
You can remove it altogether, can't you ?

Reply:
Why not using [`<oui-tooltip>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip) component instead?